src/topology/topological_fiber_bundle.lean

```diff
@@ -979,3 +1051,111 @@ begin
 end
 
 end topological_fiber_bundle_core
+
+section topological_fiber_prebundle
+
+variables (F) {Z : Type*} [topological_space B] [topological_space F] {proj : Z → B}
+
+/-- This structure permits to define a fiber bundle when trivializations are given as local
+equivalences but there is not yet a topology on the total space. The total space is hence given a
+topology in such a way that there is a fiber bundle structure for which the local equivalences
+are also local homeomorphism and hence local trivializations. -/
+@[nolint has_inhabited_instance]
+structure topological_fiber_prebundle (proj : Z → B) :=
+(trivialization_at : B → prebundle_trivialization F proj)
+(trivialization_at_mem : ∀ x : B, x ∈ (trivialization_at x).base_set)
```

Comment:
The name is not good: it's not the `trivialization_at` which belongs to something. `mem_base_trivialization_at`?